src/chrome_client.ts

```diff
@@ -139,8 +140,12 @@ export class ChromeClient {
     if (!options.hostname) {
       options.hostname = "localhost";
     }
+
     // Create the sub process
-    const chromePath = await this.getChromePath();
+    const chromePath =
+      await ((options.binaryPath)
+        ? this.getChromePath(options.binaryPath)
+        : this.getChromePath());
```

Comment:
I think we should have some consitency between how this is implemented internally for both firefox and chrome - firefox just does `const path = passedInPath || getFirefoxPath()`, where as chrome doesn't

Maybe here we could do `const chromePath = binaryPath || this.getChromePath()` as well?

Reply:
I was planning to do that at first. But then I saw that in chrome, file existence is being checked, so thought of keeping the path selection there. Will make it uniform like Firefox